tools/flaky_test_monitor/level3/flaky-test-monitor.properties

```diff
@@ -0,0 +1,21 @@
+# property file for Flaky Test Monitor
+
+##############################
+#####   LEVEL 3  #############
+##############################
+
+# includes tests that have a failure rate of at least the specified percentage
+# e.g. 0.40 == 40%, 0.75 == 75%
+failures_threshold_percent=0
```

Comment:
do we want 0 as a default here?

Reply:
Good catch. We want to filter out tests with 0% failure rate. I increased this to 0.01 (1%)

https://github.com/onflow/flow-go/pull/1793/commits/7ca42b8a930036d6e038e08ad342650a254e482a